src/Categories/Monad/Construction/Kleisli.agda

```diff
@@ -0,0 +1,116 @@
+module Categories.Monad.Construction.Kleisli where
+
+-- Definition of kleisli triple as a relative monad
+-- and the equivalence of kleisli triple and monad
+-- see https://ncatlab.org/nlab/show/extension+system (the naming differs)
+
+open import Level
+open import Categories.Category
+open import Categories.Monad.Relative renaming (Monad to RMonad)
+open import Categories.Monad
+open import Categories.Functor renaming (id to idF)
+open import Categories.NaturalTransformation renaming (id to idN)
+import Categories.Morphism.Reasoning as MR
+
+private
+    variable
+        o ℓ e : Level
+
+module _ (C : Category o ℓ e) where
+    open Category C
+    open HomReasoning
+    open Equiv
+    open MR C
+
+    -- a kleisli triple is a relative monad with J = idF
+    Kleisli : Set (o ⊔ ℓ ⊔ e)
+    Kleisli = RMonad {C = C} {D = C} idF
+
+    -- kleisli triples are equivalent to monads
+    Kleisli⇒Monad : Kleisli → Monad C
+    Kleisli⇒Monad K = record
+        { F = F
+        ; η = η
+        ; μ = μ
+        ; assoc = assoc'
+        ; sym-assoc = sym assoc'
+        ; identityˡ = identityˡ'
+        ; identityʳ = K.identityʳ
+        }
+        where
+            module K = RMonad K
+            open K using (unit; extend)
+
+            -- Extract a functor from the relative monad
+            F : Endofunctor C
+            F = RMonad⇒Functor K
+            open Functor F
+
+            -- constructing η from unit
+            η = ntHelper {F = idF} {G = F} record
+                { η = λ X → unit
+                ; commute = λ f → sym K.identityʳ
+                }
+
+            -- constructing μ from extend
+            μ = ntHelper {F = F ∘F F} {G = F} record
+                { η = λ X → extend id
+                ; commute = λ f → begin 
+                    extend id ∘ extend (unit ∘ extend (unit ∘ f))   ≈⟨ sym K.assoc ⟩
```

Comment:
This repeated use of `sym K.assoc` makes me think that `RelativeMonad` should be extended to have `sym-assoc` in its API (this is optional, but would be nice).

Reply:
Changed it for `RelativeMonad` and `RelativeComonad` and also fixed places where its used.